i18n/fr.yml

```diff
@@ -233,7 +233,6 @@ components:
     stops: Arrêts et stations
     streets: Plan des rues
   MetroUI:
-    arriveAtTime: "Arrivée à {time}"
```

Comment:
Why not also remove this key in other languages (ko, vi..)?

Reply:
Looks like someone already merged a commit into dev that took care of the other languages!